backend/github.js

```diff
@@ -96,3 +96,75 @@ exports.deleteHook = function (name, hook, accessToken, callback) {
     }
   });
 };
+
+/*
+ * hookValidator is used to check whether hook is registered to repository or not
+ * @param repositoryName: name of the repository
```

Comment:
@param name: `full_name` of the repository
Follow the style used while commenting other functions in the project